config/containers/logging/configure.md

```diff
@@ -159,25 +172,26 @@ see more options.
 
 ## Limitations of logging drivers
 
-- Users of Docker Enterprise can make use of "dual logging", which enables you to use the `docker logs`
-command for any logging driver. Refer to 
-[Reading logs when using remote logging drivers](/config/containers/logging/dual-logging/) for information about 
-using `docker logs` to read container logs locally for many third party logging solutions, including: 
-
-    - syslog	
-    - gelf	
-    - fluentd	
-    - awslogs	
-    - splunk	
-    - etwlogs	
-    - gcplogs	
-    - Logentries
-    
-- When using Docker Community Engine, the `docker logs` command is only available on the following drivers:
-
+- Users of Docker Enterprise can make use of "dual logging", which enables you
+  to use the `docker logs` command for any logging driver. Refer to
+  [reading logs when using remote logging drivers](dual-logging.md) for
+  information about using `docker logs` to read container logs locally for many
+  third party logging solutions, including: 
+    - `syslog`
+    - `gelf`
+    - `fluentd`
+    - `awslogs`
+    - `splunk`
+    - `etwlogs`
+    - `gcplogs`
+    - `Logentries`
+- When using Docker Community Engine, the `docker logs` command is only available
+  on the following drivers:
     - `local`
     - `json-file`
     - `journald`
-
-- Reading log information requires decompressing rotated log files, which causes a temporary increase in disk usage (until the log entries from the rotated files are read) and an increased CPU usage while decompressing. 
-- The capacity of the host storage where docker’s data directory resides determines the maximum size of the log file information.
+- Reading log information requires decompressing rotated log files, which causes
+  a temporary increase in disk usage (until the log entries from the rotated
+  files are read) and an increased CPU usage while decompressing. 
+- The capacity of the host storage where docker’s data directory resides
```

Comment:
How about `The capacity of the host storage where the Docker data directory resides determines the maximum size of the log file information.`

Reply:
Ah, yes, I can do that